six/modules/c++/six.sidd/include/six/sidd/Display.h

```diff
@@ -206,6 +206,9 @@ struct BandInformation
 {
     BandInformation();
 
+    size_t numBands;
```

Comment:
Can you initialize this in the constructor to the same reserved uninitialized value that `displayFlag` gets set to?

Other than that, looks good.